internal/app_test.go

```diff
@@ -73,11 +72,7 @@ func encrypt(t *testing.T, config map[string]*ConfigFile) {
 }
 
 func testWrapper(t *testing.T, doGet http.HandlerFunc, testFunc func(app *App, client *http.Client, tempdir string)) {
-	dir, err := ioutil.TempDir("", "")
-	if err != nil {
-		t.Error(err)
-	}
-	defer os.RemoveAll(dir)
+	dir := t.TempDir()
```

Comment:
Does the `TempDir` auto-cleanup the directory after the last reference to it becomes inaccessible?
The `defer os.RemoveAll(dir)` is still needed if not.

Reply:
yep. its an easier pattern to use with unit tests